nav2_smac_planner/src/analytic_expansion.cpp

```diff
@@ -189,6 +189,10 @@ typename AnalyticExpansion<NodeT>::AnalyticExpansionNodes AnalyticExpansion<Node
 
   // A move of sqrt(2) is guaranteed to be in a new cell
   static const float sqrt_2 = std::sqrt(2.0f);
+  if (d < sqrt_2) {
```

Comment:
Add it to the `if` statement above 

```
if (d > _search_info.analytic_expansion_max_length || d < sqrt_2) {
```

Reply:
^ that would require moving the sqrt_2 definition above as well

Reply:
Thanks for feedback!